src/components/ArpSpoof/ArpSpoof.tsx

```diff
@@ -49,7 +50,7 @@ const ARPSpoofing = () => {
         "Step 4: View the output block below to view the results.";
 
     // Component Constants.
-    const dependencies = ["dsniff"]; // Contains the dependencies required for the component.
+    const dependencies = ["arpspoof"]; // Contains the dependencies required for the component.
```

Comment:
Sorry @vtruglio. this is my bad. I was confused about what the dependencies variable should store, but I have subsequently tested it and can confirm that it should be the name of the package that requires installation. Therefore, please change back to dsniff. 
```suggestion
    const dependencies = ["dsniff"]; // Contains the dependencies required for the component.
```